register.js

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
If Javascript does not care about semi colons, that does not mean that they should not be used

Reply:
I have read about this topic, it was said that a consistent coding style makes code more readable. so i have to decide whether i will or won’t use semicolons, and do so everywhere.